src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/NonEmptyAtclauseDescriptionCheck.java

```diff
@@ -54,6 +54,11 @@
     }
 
     @Override
+    public int[] getRequiredJavadocTokens() {
+        return getDefaultJavadocTokens();
+    }
```

Comment:
> the whole reason of the issue was to customize NonEmptyAtclauseDescription 

I took your reply [here](https://github.com/checkstyle/checkstyle/pull/3512#issuecomment-255626830) to mean there were no optional tokens for any checks.

> > Was that documentation valid where the 2 tokens are optional?

> I doubt it was valid. It was imaginary config. 

> > Are there any other optional tokens for the javadoc checks?

> We do not have.

Done in next update.